ci/licenses_golden/licenses_flutter

```diff
@@ -936,7 +936,7 @@ TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
 
    "Source" form shall mean the preferred form for making modifications,
    including but not limited to software source code, documentation
-   source, and configuration files.
+   source, and configuration  .
```

Comment:
The license bot is complaining that this change is dubious. Not sure how though since the file should be generated. Perhaps there was an edit before a save?